src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import NavbarElement from "./NavbarElement";
+
+class Navbar extends Component {
+
+    constructor() {
+        super();
+        this.state = {
+            menuOpened: false,
+            elements: [
+                {
+                    name: "Home",
+                    linksTo: "/"
+                },
+                {
+                    name: "Documentation",
+                    linksTo: "/docs"
+                },
+                {
+                    name: "News",
+                    linksTo: "/news"
+                },
+                {
+                    name: "Contribute",
+                    linksTo: "/contribute"
+                }
+            ]
+        }
+    }
+
+    handleClick = () => {
+        this.setState(state => ({
+            menuOpened: !state.menuOpened
+        }))
+    }
+
+    render() {
+        let menuConditionalclassNames = this.state.menuOpened ? "" : "hidden"
+
+        return (
+            <nav className="flex justify-between flex-wrap py-5 px-10 md:px-12 lg:px-36 xl:px-46 2xl:px-64 bg-gray-700 text-gray-100">
```

Comment:
@RubyNova - need / want your input on this.

![image](https://user-images.githubusercontent.com/10300290/104831037-e8ba0600-5852-11eb-9be8-aaa193e18f7a.png)

Currently, the background is outputting the color on the left - the color on the right is what was originally mocked in Figma.... 
Which do you like more? So that we can keep the design consistent :D

Reply:
@capnkenny @RubyNova The reason the colour is different is because we've decided to go with TailwindCSS, I thought it would make sense to make use of their colour palette (you can see it here: https://tailwindcss.com/docs/customizing-colors). By default it uses `coolGray` as `gray`. I think if you're not satisfied we could enable their `trueGray` in the config which I think looks closer to what is in figma - you can view it at https://tailwindcss.com/docs/customizing-colors#color-palette-reference . I think sticking to a colour from their palette is a good choice as they come with nice pre-made intensity levels like 50, 100, 200 etc. (see the palette I've linked earlier). Please look at the various gray types they have - coolGray, trueGray, warmGray etc. and let me know what you think.

Reply:
Keeping in mind that we can always revert to custom colours if needed, I think we're going to stick with the Tailwind option for now. @RubyNova - Figma should be updated with the colour so you can at least get an idea on colour scheme if you're interested. (I've also switched to UK spelling so that I'm consistent with you guys :D)